src/runtime-rs/crates/runtimes/src/manager.rs

```diff
@@ -106,6 +122,20 @@ impl RuntimeHandlerManagerInner {
         };
 
         let config = load_config(spec, options).context("load config")?;
+
+        // note that trace_setup() will only be called when enable_tracing is true
+        if config.runtime.enable_tracing
+            && trace_setup(
+                &self.id,
+                &config.runtime.jaeger_endpoint,
+                &config.runtime.jaeger_user,
+                &config.runtime.jaeger_password,
+            )
+            .is_ok()
```

Comment:
What if the `trace_setup()` returns an error, raises the error, or maybe prints to log?

Reply:
Fixed.